src/broker/cas_cgw.c

```diff
@@ -1227,6 +1252,17 @@ cgw_set_bindparam (T_CGW_HANDLE * handle, int bind_num, void *net_type, void *ne
       data_size = 0;
     }
 
+  // Oracle ODBC does not support the BIGINT type.
+  // So, change it to Numeric type.
+  if (curr_dbms_type == SUPPORTED_DBMS_ORACLE)
```

Comment:
what about remove inner-if by changing if condition

` if (curr_dbms_type == SUPPORTED_DBMS_ORACLE && type == CCI_U_TYPE_BIGINT)`

Reply:
Thanks for the advice.